### ISSUE TITLE
Use a wider aspect ratio for large cards on mobile

### ISSUE THREAD

**Issue by danrabbit** [elementary/blog-template#58]
## Problem

On mobile we have to do so much scrolling because of how tall all the cards are. I opened another issue about a tighter layout for small cards, but the large cards are still very tall

## Proposal

Another small change we could make is using a 16:9 aspect ratio for the large card previews on mobile. This would also just be more like the desktop site where we use a wider card aspect ratio

**Comment by cassidyjames** [comment]
Yeah I think we should just stick to 16:9 for all the cards for a consistent look; that's what Twitter does as well, so we have to format them that way, anyway.
